cpp/benchmarks/bench_comm.cpp

```diff
@@ -263,6 +279,17 @@ int main(int argc, char** argv) {
     // We start with disabled statistics.
     auto stats = std::make_shared<rapidsmpf::Statistics>(/* enable = */ false);
 
+#ifdef RAPIDSMPF_HAVE_CUPTI
+    // Create CUPTI monitor if enabled
+    std::unique_ptr<rapidsmpf::CuptiMonitor> cupti_monitor;
+    if (args.enable_cupti_monitoring) {
+        cupti_monitor = std::make_unique<rapidsmpf::CuptiMonitor>();
+        cupti_monitor->start_monitoring();
+        log.print("CUPTI memory monitoring enabled");
+    }
+#endif
+
```

Comment:
```suggestion
```

Reply:
Also done in https://github.com/rapidsai/rapidsmpf/pull/445/commits/d7449f3ad57fc2f5c2d1038e36e0f1f40e34a8d4, but the GH preview considers your suggestion for line 291 and the diff shows the line 292 being deleted, therefore it doesn't resolve the suggestion. 🤣